# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchDriver.cpp
# Commit: 32f68cf9630d
# Full Hash: 32f68cf9630dfc650bcef2e2e82798b563b712ad
# Author: Frédéric Wang <fwang@igalia.com>
# Date: 2024-06-12 15:00:37
# Regressor Bug: 1351231
# File Overlap Count: 1
# Description:
#   Bug 1881040 - Propagate InterceptedHttpChannel information when using PFetch. r=edenchuang
#   
#   This interception information was introduced in bug 1658869 and is used
#   when calling fetch from the main thread. However, after bug 1351231 we
#   are not propagating that information when using PFetch, because it is
# ==============================================================================

diff -r 7d3e64562102 -r 32f68cf9630d dom/fetch/FetchDriver.cpp
--- a/dom/fetch/FetchDriver.cpp	Wed Jun 12 07:50:40 2024 +0000
+++ b/dom/fetch/FetchDriver.cpp	Wed Jun 12 07:50:40 2024 +0000
@@ -675,7 +675,7 @@
   }
 
   // If the fetch is created by FetchEvent.request or NavigationPreload request,
-  // corresponding InterceptedHttpChannel information need to propagte to the
+  // corresponding InterceptedHttpChannel information need to propagate to the
   // channel of the fetch.
   if (mRequest->GetInterceptionTriggeringPrincipalInfo()) {
     auto principalOrErr = mozilla::ipc::PrincipalInfoToPrincipal(